contrib/containerlab/bgpv2/auto-discovery/default-gateway/multi-homing/README.md

```diff
@@ -0,0 +1,165 @@
+Multi-homing BGP with Default Gateway Auto Discovery
+====================================================
+
+Multi-homing BGP means Cilium node is peering with two upstream routers over different links.
+
+Common design requirements
+
+- Node is connected to two different top-of-rack switches for redundancy.
+- Separation of east-west and north-south traffic via different links on the node. This may require different advertisements to each peer.
+
+BGP Resources
+-------------
+
+**BGP instance configuration**
+
+In the below example, we define two eBGP peers without peer address. Peer address will be auto discovered using default gateway
+for each address family. It will create bgp session per peer configured so each address family will have one bgp session  , both of which have identical peering configuration defined in
+CiliumBGPPeerConfig resource with name `cilium-peer`.
+
+```yaml
+  bgpInstances:
+  - name: "65001"
+    localASN: 65001
+    peers:
+    - name: "65000 ipv6"
```

Comment:
Nit: We should validate this properly, but peer name with space is not preferable. Let's go with `65000-ipv6` or something.

Reply:
fixed